transports/http/endpoints/api/merkleroots/endpoints.go

```diff
@@ -0,0 +1,59 @@
+package merkleroots
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/libsv/bitcoin-hc/service"
+	router "github.com/libsv/bitcoin-hc/transports/http/endpoints/routes"
+)
+
+type handler struct {
+	service service.Headers
+}
+
+// NewHandler creates new endpoint handler.
+func NewHandler(s *service.Services) router.ApiEndpoints {
+	return &handler{service: s.Headers}
+}
+
+// RegisterApiEndpoints registers routes that are part of service API.
+func (h *handler) RegisterApiEndpoints(router *gin.RouterGroup) {
+	merkle := router.Group("/chain/merkleroots")
```

Comment:
Not true.
We have, for example:
`/chain/tips/`
`/network/peers`

I think that since we pass multiple merkle roots, it makes sense to call the path `merkleroots`. But if you insist on changing it, I will :)

Reply:
Ohhh so we have such an inconsistence,
so we should probably discuss this at the channel, and add a task for unification of the paths,
and actually if we choose the pattern as a team, then you could instantly adjust this endpoint to it.

Reply:
Sure, if we decide on one approach, I can change all the endpoints